README.md

```diff
@@ -130,7 +130,12 @@ Prerequisites for running browser-based tests:
    ```
 4. If a virtual environment is used to run openwisp-radius then
    this needs to be activated before running browser tests.
-5. In the test environment of openwisp-radius, the `default` organization
+5. Configuration file of `mobile` organization is needed before running `yarn start`.
+   `mobile` organization can be created by running:
+   ```
+   yarn add-org --noprompt '{"name":"mobile","slug":"mobile","uuid":"organization_uuid","secret_key":"organization_secret_key","mobile_phone_verification":true,"subscriptions":false,"login_action_url":"http://localhost:8000/captive-portal-mock/login/","logout_action_url":"http://localhost:8000/captive-portal-mock/logout/","logout_by_session_ID":true,"remember_me":true,"openwisp_radius_url":"http://127.0.0.1:8000","assets_confirm":true}'
+   ```
+6. In the test environment of openwisp-radius, the `default` organization
    must be present or alternatively
```

Comment:
BTW I noticed an unfinished sentence here, can you please remove the unfinished part?